web/client/components/map/leaflet/plugins/GoogleLayer.js

```diff
@@ -8,7 +8,10 @@
 
 var Layers = require('../../../../utils/leaflet/Layers');
 var Google = require('leaflet-plugins/layer/tile/Google');
-
+Google.prototype._checkZoomLevels = function() {
+    // Avoid map zoom setting when current zoom is greatr then  the google's max zoom
+};
 Layers.registerType('google', (options) => {
-    return new Google(options.name, {zoomOffset: options.zoomOffset || 0});
+    return new Google(options.name, {zoomOffset: options.zoomOffset || 0, maxNativeZoom: options.maxNativeZoom || 18,
+        maxZoom: options.maxZoom || 20});
```

Comment:
Shouldn't this be 23?

Reply:
Google layer is the only one that will not zoom more then 20. So here we have to decide if is better to show wrong tile, (set maxZoom = 23) or nothing (maxZoom= 20).
Let me know what you prefer.

Reply:
Ok, I think 20 is fine, then